.travis.yml

```diff
@@ -0,0 +1,10 @@
+language: php
```

Comment:
After discussion with Ken I'm leaning toward developing this in Python (CherryPy and Jinja) so this file should be updated to reflect that